resources/application-connector/charts/central-application-connectivity-validator/templates/autoscaling.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.autoscaling.enable }}
+{{- if and (.Values.autoscaling.enable) (.Values.global.disableLegacyConnectivity) }}
```

Comment:
Actually you could use `disableLegacyConnectivity` flag in `resources/application-connector/Chart.yaml` and get rid of `centralApplicationConnectivityValidatorEnabled`.